pkg/common/httpx/httpx.go

```diff
@@ -59,7 +60,7 @@ func DownloadWithOpts(opts DownloadOpts) error {
 }
 
 func DownloadWithChanged(opts DownloadOpts) (bool, error) {
-	if pathx.Exists(opts.File) {
+	if pathx.Exists(opts.File) && !opts.Override {
```

Comment:
override flag will force to always download so that whole method `DownloadWithChanged` does not make sense